components/ClaimCard.tsx

```diff
@@ -1,18 +1,19 @@
-import Link from 'next/link'
-import { useTranslation } from 'next-i18next'
-import { useRouter } from 'next/router'
-
-import { Button } from './Button'
+import { ClaimCardDetails } from './ClaimCardDetails'
+import { ClaimCardBody } from './ClaimCardBody'
 
 export const ClaimCard: React.FC = () => {
-  const { t } = useTranslation('common')
-  const router = useRouter()
-
   return (
     <div className="claim-card">
-      <Link href="/" locale={router.locale === 'en' ? 'es' : 'en'}>
-        <Button primary label={t('change-locale')} />
-      </Link>
+      <ClaimCardDetails title="Unemployment Insurance (UI)" benefitYear="3/21/2020 - 3/20/2021" claimBalance="$0.00" />
```

Comment:
Good placeholders, but do we need a ticket for an app-level state object? Not exactly about this PR, just a thought.

Reply:
dunno! how do you mean?